Archs/MIPS/MipsParser.cpp

```diff
@@ -58,6 +71,15 @@ const MipsRegisterDescriptor mipsPs2Cop2FpRegisters[] = {
 	{ L"vf30", 30 },	{ L"vf31", 31 },
 };
 
+const MipsRegisterDescriptor mipsRspCop0Registers[] = {
+	{ L"sp_mem_addr", 0 },	{ L"sp_dram_addr", 1 }, { L"sp_rd_len", 2 },
+	{ L"sp_wr_len", 3 },	{ L"sp_status", 4 },	{ L"sp_dma_full", 5 },
+	{ L"sp_dma_busy", 6 },	{ L"sp_semaphore", 7 },	{ L"dpc_start", 8 },
+	{ L"dpc_end", 9 },		{ L"dpc_current", 10 },	{ L"dpc_status", 11 },
+	{ L"dpc_clock", 12 },	{ L"dpc_bufbusy", 13 },	{ L"dpc_pipebusy", 14 },
+	{ L"dpc_tmem", 15 },
+};
+
```

Comment:
What would happen if you tried to access a register above $15? If it leads to an exception, then parseRegisterNumber should probably not allow those higher numbers either.